Rules4Net.Tests/Constraints/BetweenConstraintTests.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Rules4Net.Data;
```

Comment:
Where are the edges cases, like tests with:
- null data, 
- upper bound null, 
- lower bound null 
- with lower bound and upper bound with diferent types